README.rst

```diff
@@ -48,6 +48,10 @@ The development version can be installed from Github ::
 
     $ pip install https://github.com/mher/flower/zipball/master#egg=flower
 
+Installing `flower` with `poetry <https://python-poetry.org/>` ::
+
+    $ poetry install
```

Comment:
This won't install the package. I think you're looking for `poetry add flower && poetry install`.

Reply:
You are right!